docs/source/keywords.rst

```diff
@@ -487,14 +487,30 @@ If cross-compiling, or if you want to be specific, the CPU architecture can be p
     * - ``Kokkos_ARCH_KNL``
       - Knights Landing/x86-64
       - 31S1P @ Tianhe-2
-
     * - ``Kokkos_ARCH_KNC``
       - Knights Corner/x86-64
       -
 
     * - ``Kokkos_ARCH_SNB``
       - Sandy Bridge/x86-64
       -
+      
+ .. list-table:: RISC-V CPU architectures
+    :widths: 30 30 30
+    :header-rows: 1
+    :align: left
+    
+    * - CMake keyword
+      - Architecture/Instruction set
+      - Examples   
+    
+    * - ``KOKKOS_ARCH_RISCV_RVA22V``
+      -  RVA22V/RISC-V ISA
+      -
```

Comment:
No examples?

Reply:
and could you add a 4th column with the kokkos version that it was added in? see https://github.com/kokkos/kokkos-core-wiki/blob/main/docs/source/keywords.rst#architectures

Reply:
I added the 4th column with the Kokkos version.

Reply:
@dalg24 What should I put as examples? Just the hardware with this CPU?

Reply:
```suggestion
      - SpacemiT K1
```